modules/pel/peltool/peltool.py

```diff
@@ -600,6 +603,10 @@ def main():
             os.remove(args.file)
         sys.exit(0)
 
+    if inBMC:
```

Comment:
I think we should move this into the `else` portion of the `if` condition (LN610) below to keep all environment-specific code in a common place. If it continues to grow, we could consider creating a utility API "setup()" in the future.